examples/all.yaml

```diff
@@ -1,5 +1,20 @@
 ---
 
+## ocp-scheduler vars
+scheduler_role_enable: false
+configure_scheduler_profile: false
+scheduler_profile: "HighNodeUtilization"
+scheduler_e2e_test_run: false
+scheduler_workdir: ~/ocp-scheduler
+scheduler_subdir_list: ["script", "e2e-test"]
+scheduler_e2e_github_repo: "https://github.com/openshift/openshift-tests-private"
+scheduler_e2e_github_branch: "master" 
```

Comment:
not used anywhere

Reply:
Used in the latest changes to specify git branch.